addons/account/wizard/account_resequence_views.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
+        <record id="account_resequence_view" model="ir.ui.view">
+            <field name="name">Re-sequence Journal Entries</field>
+            <field name="model">account.resequence.wizard</field>
+            <field name="arch" type="xml">
+            <form string="Re-Sequence">
+                <field name="move_ids" invisible="1"/>
+                <field name="new_values" invisible="1"/>
+                <field name="sequence_number_reset" invisible="1"/>
+                <group>
+                    <group>
+                        <field name="ordering" widget="radio"/>
+                    </group>
+                    <group>
+                        <field name="first_name"/>
+                    </group>
+                </group>
+                <label for="preview_moves" string="Preview Modifications"/>
+                <field name="preview_moves" widget="account_resequence_widget" class="d-block"/>
```

Comment:
class="d-block" ?

Reply:
bootstrap for
```css
.d-block {
    display: block !important;
}
```
It is used to take the whole width of the wizard